src/main/java/org/Tema10/FestivalApp.java

```diff
@@ -0,0 +1,39 @@
+package org.Tema10;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class FestivalApp {
+
+    public static void startFestival(int numberOfAttendees) {
+        FestivalGate gate = new FestivalGate();
+
+        FestivalStatisticsThread statsThread = new FestivalStatisticsThread(gate);
+        statsThread.setDaemon(true);
+        statsThread.start();
+
+        for(int i = 0; i < numberOfAttendees; i++) {
+            TicketType ticketType = getRandomTicketType();
+            new FestivalAttendeeThread(ticketType, gate).start();
+        }
+
+        FestivalApp.keepAppRunning(5100);
```

Comment:
can't it simply be 
```suggestion
        keepAppRunning(5100);
```
?
it's just a static method inside the same class